corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -81,7 +82,10 @@ def __init__(self, suite, module, detail_section_elements):
         self.domain = self.app.domain
         self.module = module
         self.detail_section_elements = detail_section_elements
-        self.case_session_var = self.module.search_config.case_session_var
+        if self.is_multi_select():
+            self.case_session_var = "search_cases"
```

Comment:
nit: I think `selected_cases` is a better description of this
```suggestion
            self.case_session_var = "selected_cases"
```